src/univariate/continuous/exponential.jl

```diff
@@ -60,6 +60,11 @@ kurtosis(::Exponential{T}) where {T} = T(6)
 
 entropy(d::Exponential{T}) where {T} = one(T) + log(d.θ)
 
+function kldivergence(p::Exponential, q::Exponential)
+    λq_over_λp = scale(q) / scale(p)
+    return -log(λq_over_λp) + λq_over_λp - 1
```

Comment:
It seems here we can actually use `logmxp1`:
```suggestion
    return -logmxp1(λq_over_λp)
```